apollo/lib/schemas/agent.py

```diff
@@ -26,3 +27,8 @@ class AgentSchema(ORMBase):
     id: uuid.UUID
     name: str
     oauth_client: OAuthClientSchema
+    connection_state: WebSocketState
+
+    @validator('connection_state')
+    def parse_to_string(cls, v):
```

Comment:
Missing class method annotation, which is what Codacy is pointing out.

Also untested.